src/libdecaf/src/cafe/libraries/snd_user/snd_user_fx.cpp

```diff
@@ -0,0 +1,43 @@
+#include "snd_user.h"
+#include "snd_user_fx.h"
+#include "cafe/libraries/cafe_hle_stub.h"
+
+namespace cafe::snd_user
+{
+
+static AXFXAllocFuncPtr
```

Comment:
One day we will support process switching so we should not use host level static variables, but instead store them in the rpl .data section. The RegisterDataInternal allows us to swap the host pointer when the userland process changes, which is actually what the call to `coreinit::internal::relocateHleLibrary` is doing in the rpl_entry - one day we will also call that on process switch.

See how we do static data in other rpls:
https://github.com/decaf-emu/decaf-emu/blob/master/src/libdecaf/src/cafe/libraries/coreinit/coreinit_alarm.cpp#L23-L41

```
struct StaticFxData
{
be2_val<AXFXAllocFuncPtr> allocFuncPtr;
be2_val<AXFXFreeFuncPtr> freeFuncPtr;
};

static virt_ptr<StaticFxData>
sFxData = nullptr;
```

https://github.com/decaf-emu/decaf-emu/blob/master/src/libdecaf/src/cafe/libraries/coreinit/coreinit_alarm.cpp#L576
`RegisterDataInternal(sFxData);`